components/shopify_partner/sources/new_app_charges.js

```diff
@@ -0,0 +1,67 @@
+const shopify = require("../shopify_partner.app.js");
+const getAppTransactions = require("../queries/getAppTransactions");
+
+module.exports = {
+  key: "shopify_partner-new-app-charges",
+  name: "New App Charges",
+  type: "source",
+  version: "0.0.1",
+  description:
+    "Emit new events when new app charges made to your partner account.",
+  props: {
+    shopify,
+    db: "$.service.db",
+    createdAtMin: {
+      type: "string",
+      description:
+        "Only include transactions after this specific time (ISO timestamp)",
+      label: "createdAtMin",
+    },
+    createdAtMax: {
+      type: "string",
+      description:
+        "Only include transactions up to this specific time (ISO timestamp)",
+      label: "createdAtMin",
+    },
+    timer: {
+      description: "How often this action should run",
```

Comment:
When I add descriptions to timers, I like to avoid Pipedream terms as much as possible (and this is a source), so I'd say something like "How often to poll for new app charges"

Reply:
If you move into a common file, I guess you could keep the description generic to all sources, e.g. "How often to poll for new events" or something like that